src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Windows.cs

```diff
@@ -0,0 +1,2459 @@
+﻿using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.Drawing.Internal;
+using System.Drawing.Text;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Drawing
+{
+    internal partial class SafeNativeMethods
+    {
+        internal partial class Gdip
+        {
+            [DllImport("kernel32")]
+            private static extern IntPtr LoadLibrary(string fileName);
+            [DllImport("kernel32")]
+            private static extern IntPtr GetProcAddress(IntPtr module, string procName);
```

Comment:
Thanks, I will relocate these (or re-use the existing definition if it's already there).